tests/ReflectionParameterTest.php

```diff
@@ -171,6 +171,21 @@ public function testParamBuiltInClassConst()
         $this->assertSame('DateTime::ATOM', $parameters[0]->getDefaultValueConstantName());
     }
 
+    public function testParamGlobalNsClassConst()
+    {
+        $this->setUpFile(__DIR__ . '/Stub/FileWithGlobalNamespacesClasses.php');
+        $parsedNamespace = $this->parsedRefFile->getFileNamespace('');
+        $parsedClass     = $parsedNamespace->getClass(\AnotherGlobalNsClass::class);
+        $parsedFunction  = $parsedClass->getMethod('foo');
+
+        $parameters = $parsedFunction->getParameters();
+        $this->assertTrue($parameters[0]->isDefaultValueConstant());
```

Comment:
Please specify message to each of the `assert...` method calls in this test, so that we can see which of two assertions failed based on the error message, because they're comparing the same things now.